test/functional/api/cas/casadm_parser.py

```diff
@@ -15,14 +15,17 @@
 from datetime import timedelta
 from typing import List
 
-from packaging import version
-
 from api.cas import casadm
 from api.cas.cache_config import *
 from api.cas.casadm_params import *
 from test_utils.size import parse_unit
 
 
+class Version:
+    def __init__(self, version_string):
+        self.main, self.major, self.minor, self.pr, self.type = version_string.split('.')
+        self.base = f"{self.main}.{self.major}.{self.minor}"
+
```

Comment:
Please look at the CasVersion class in /api/cas/version.py and consider merging it with your Version class

Reply:
Done.